hug/output_format.py

```diff
@@ -75,6 +76,8 @@ def _json_converter(item):
         return str(item)
     elif isinstance(item, timedelta):
         return item.total_seconds()
+    elif isinstance(item, UUID):
```

Comment:
Might be better to combine this with Decimal check, e.g. `isinstance(item, (Decimal, UUID))`